templates/includes/pagination.html

```diff
@@ -0,0 +1,13 @@
+{% if is_paginated %}
+  {% if is_paginated %}
```

Comment:
The check for `is_paginated` is repeated. You can remove the second `{% if is_paginated %}` on line 2 to simplify the code.